cli/simulate_pixels.py

```diff
@@ -670,6 +678,16 @@ def save_results(event_times, results, i_trig, i_mod=-1, light_only=False):
             response = cp.load(response_file[i_mod-1])
             RangePop()
 
+            RangePush("load_pixel_thresholds")
```

Comment:
Same questions regarding a non-specified pixel_threshold_file. I think it would be useful to support defaults.